### PR TITLE
Feature/web matrics

### PR DESCRIPTION
**Pull request by roniy68**
# Highlights 🥇 
---

## What I Have Done: 
✅ API  : selected -  https://restcountries.com/v3.1/
✅ Design:

- Follw these [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), including:
- Colors (select one main color for your website).
- Typography: font face, size and weight.
- Layout: composition and space between elements.
- Original design idea by [Nelson Sakwa on Behance](https://www.behance.net/sakwadesignstudio).
✅ Home page
✅ Details page
✅ Create unit tests for pure functions in your Redux code.
- Create integration tests for your application using the React Testing Library.
- Mock the access to the API, so that your tests don't send actual requests.
✅ The project is a single-page application (SPA) built with React and Redux.

# Project Documentaion : 
---
### Changes details

  ...
## Missed anything?
- [ ] Explained the purpose of this PR.
- [ ] Self reviewed the PR.
- [ ] Added or updated test cases.
- [ ] Informed of breaking changes, testing and migrations (if applicable).
- [ ] Updated documentation (if applicable).
- [ ] Attached screenshots (if applicable).

## Followed Rules: 
- Made sure [no linter errors](https://github.com/microverseinc/linters-config)
-  Followed [Github Flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md)
-  Documented [professionaly](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)
-  Followed the list of [best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).
-  Followed our list of [best practices for JavaScript](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -149,7 +149,6 @@ I am always looking for ways to improve my project. If you have any suggestions
 ## 📧 Authors <a name="authors"></a>
 
 - ***[Ahmed Hasan Rony](https://github.com/roniy68)***
-- ***[Nwadinigwe Victor chiemerie](https://github.com/Victor-chiemerie)***
 
 ## 🤝 Contributing <a name="contributing"></a>
```
Comment (iambenkis) [inline README.md:154]: - [x] - [x] Good job adding the README that contains an "Acknowledgements" section 👏 , however you should gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/). Kindly  add in your  "Acknowledgements" section  the  appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/). 👨‍🍳

File: src/__test__/components/Navbar.test.js [hunk 2]
```diff
@@ -1,17 +0,0 @@
...
-    const tree = renderer
-      .create(
-        <BrowserRouter>
-          <Navbar />
-        </BrowserRouter>,
-      )
-      .toJSON();
-    expect(tree).toMatchSnapshot();
-  });
-});
```
Comment (iambenkis) [inline src/__test__/components/Navbar.test.js:17]: - [x] Keep doing the great work, All your testes are passing 👍

**Review by iambenkis — CHANGES_REQUESTED** [review]
Hi @roniy68 ,


Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (roniy68) [reply README.md:154]: ### 😄 I have Added The acknowledgment. Sorry for this silly mistake.

File: src/__test__/components/Navbar.test.js (hunk 2 above)
Comment (roniy68) [reply src/__test__/components/Navbar.test.js:17]: # 🛩️ Thank You So Much 

### Happy Coding 😄

File: README.md (hunk 1 above)
Comment (peterrobert) [reply README.md:154]: This change has been done well

**Review by peterrobert — APPROVED** [review]
Hi @roniy68 , 


### Highlights!

[ ] Good feature branch name ✔️
[ ] Descriptive pull request ✔️


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or tag me @peterrobert if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**